ts/Series/Bellcurve/BellcurveSeries.ts

```diff
@@ -164,6 +164,11 @@ class BellcurveSeries extends AreaSplineSeries {
         if (series.baseSeries?.yData?.length || 0 > 1) {
             series.setMean();
             series.setStandardDeviation();
+
+            if (series.points?.length && series.points[0]?.destroyed) {
+                series.points = series.data;
+            }
+
```

Comment:
I think there is a possibility for a much easier solution. The issue is based on that, setData tries to update points, which don't exist (are destroyed). I've just tested and if we set a 4th flag (`updatePoints`) as false for `setData` in this method (`setDerivedData`) then everything works as expected.

Reply:
Alright, good catch!